docs/micro-rdk/movement-sensor/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Micro-RDK Movement Sensor Models"
+linkTitle: "Movement Sensor"
+weight: 30
+type: "docs"
+description: "Movement Sensor support in the micro-rdk."
+images: ["/icons/components/imu.svg"]
+tags: ["movement sensor", "components", "micro-rdk"]
+# SMEs: Nick M., Gautham V.
+---
+
+## Configuration
+
+The micro-RDK currently supports the following models of Movement Sensor:
+
+<!-- prettier-ignore -->
+| Model                             | Description                                                                                         |
```

Comment:
Looks like prettier got to it before the ignore was put in, if you care to delete the extra lines and spaces :D

Reply:
done!